csharp/Microsoft.Azure.Databricks.Client/IPermissionsApi.cs

```diff
@@ -0,0 +1,369 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Databricks.Client
+{
+    public interface IPermissionsApi : IDisposable
+    {
+        /// <summary>
+        /// Asynchronously returns a list of <see cref="AclPermissionDescription"/> representing all possible
+        /// <see cref="PermissionLevel"/> values that can be applied to tokens
+        /// </summary>
+        Task<IEnumerable<AclPermissionDescription>> GetTokenPermissionsLevels(CancellationToken cancellationToken = default);
```

Comment:
```suggestion
        Task<IEnumerable<AclPermissionDescription>> GetTokenPermissionLevels(CancellationToken cancellationToken = default);
```
Keep the naming convention consistent.